proto/quicksilver/airdrop/v1/query.proto

```diff
@@ -14,6 +18,26 @@ service Query {
     option (google.api.http).get =
         "/quicksilver/airdrop/v1/params";
   }
+  // ZoneDrop returns the details of the specified zone airdrop.
+  rpc ZoneDrop(QueryZoneDropRequest) returns (QueryZoneDropResponse) {
+    option (google.api.http).get =
+        "/quicksilver/airdrop/v1/zonedrop";
+  }
+  // ZoneDrops returns all zone airdrops of the specified status.
+  rpc ZoneDrops(QueryZoneDropsRequest) returns (QueryZoneDropsResponse) {
+    option (google.api.http).get =
+        "/quicksilver/airdrop/v1/zonedrops";
+  }
+  // ClaimRecord returns the claim record that corresponds to the given zone and address.
+  rpc ClaimRecord(QueryClaimRecordRequest) returns (QueryClaimRecordResponse) {
+    option (google.api.http).get =
+        "/quicksilver/airdrop/v1/claimrecord";
+  }
+  // ClaimRecords returns all the claim records of the given zone.
+  rpc ClaimRecords(QueryClaimRecordsRequest) returns (QueryClaimRecordsResponse) {
+    option (google.api.http).get =
+        "/quicksilver/airdrop/v1/claimrecords";
```

Comment:
I think this needs to be `"/quicksilver/airdrop/v1/claimrecords/{chain_id}/{address}"`